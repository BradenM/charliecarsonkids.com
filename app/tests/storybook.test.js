/**
 *
 * Init Storybook Snapshot Tests
 *
 */

import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();
