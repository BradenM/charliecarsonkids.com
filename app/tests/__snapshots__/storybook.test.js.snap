// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots A default 1`] = `
<a
  className="A-br8h0y-0 dwVNvo"
  href="https://www.google.com"
>
  A Link
</a>
`;

exports[`Storyshots Img default 1`] = `
<img
  alt="placeholder - 200x200"
  src="https://via.placeholder.com/200"
/>
`;
